pkg/reconciler/pipelinerun/pipelinerun_test.go

```diff
@@ -2535,11 +2541,11 @@ status:
 					Status:           &v1beta1.TaskRunStatus{},
 				},
 			},
-			expectedEvents:       []string{"Warning Failed PipelineRun \"test-pipeline-run-stopped-run-finally\" was cancelled"},
-			hasNilCompletionTime: false,
-			isFailed:             true,
-			trInStatusCount:      1,
-			skippedTasks:         []v1beta1.SkippedTask{{Name: "hello-world-2", Reason: v1beta1.GracefullyStoppedSkip}},
+			expectedEvents:        []string{"Warning Failed PipelineRun \"test-pipeline-run-stopped-run-finally\" was cancelled"},
+			hasNilCompletionTime:  false,
+			isFailed:              true,
+			childRefInStatusCount: 1,
```

Comment:
For such test cases that check the expected number of taskruns/ runs in the reconciled taskruns, I have been wondering if the changing of childReferences as the expected number could fully cover the expected test cases?

Reply:
cc @abayer would you mind checking on the change of the ChildReferences logics for the unit tests? also at https://github.com/tektoncd/pipeline/pull/5934#issuecomment-1382230129

Reply:
Yeah, I think that’s probably fine. Theoretically, checking the number of `TaskRun`s and the number of `Run`s independently is a bit more thorough, but we’re testing that the right types are created elsewhere, so I don’t see a problem in just doing a straight count of `ChildReferences` in these cases.

Reply:
Ah, I misunderstood - ignore me, listen to Lee. =)